core/src/main/java/org/elasticsearch/index/get/ShardGetService.java

```diff
@@ -238,7 +223,7 @@ public GetResult innerGet(String type, String id, String[] gFields, boolean real
                             value = source.source.length();
                         } else {
                             if (searchLookup == null) {
-                                searchLookup = new SearchLookup(mapperService, fieldDataService, new String[]{type});
+                                searchLookup = new SearchLookup(mapperService, null, new String[]{type});
```

Comment:
This is for document GET right? So it shouldn't be using fielddata, only translog or stored fields?